components/screens/deliveries/EditDelivery.tsx

```diff
@@ -890,6 +914,46 @@ class EditDelivery extends React.Component<Props, State> {
       </View>
     );
   }
+
+  /**
+   * Method for rendering fresh product prices
+   */
+   private renderFreshProductQualityPrices = () => {
+    const { itemGroupCategory } = this.props;
+    const { deliveryQualities, productPrice, productId } = this.state;
+
+    if (itemGroupCategory !== ItemGroupCategory.FRESH || !deliveryQualities.length) {
+      return null;
+    }
+
+    return (
+      <>
+        <Text style={ styles.smallHeader }>
+          Mahdolliset laatubonukset alla, lopullinen laatuluokka varmistuu vastaanoton yhteydessä
+        </Text>
+        {
+          deliveryQualities.map((deliveryQuality, index) => {
+            const name = deliveryQuality.displayName;
+            const priceBonus = deliveryQuality.priceBonus;
+            const priceBonusVAT = roundPrice(priceBonus * 1.14);
+            const unit = productPrice?.unit.toUpperCase();
+            const active = deliveryQuality.deliveryQualityProductIds.some(id => id === productId);
+
+            if (!active) {
+              return null;
+            }
+
+            return (
+              <Text key={ index } style={ styles.listItem }>
+                { `${ name } ${ priceBonus } €/kg ALV 0% (${ priceBonusVAT } €/${ unit } ALV 14%)` }
```

Comment:
Unit-property is used only in ALV 14% part, shouldn't it be used in ALV 0% part too?

Reply:
Unit removed from bonus prices completely